frontend/src/App.js

```diff
@@ -1,48 +1,24 @@
 import React, { Component } from 'react'
-import './App.css'
+import Header from './Header'
+import Navigation from './Navigation'
+import Section from './Section'
+import Footer from './Footer'
 
 class App extends Component {
     constructor(props) {
-      super(props)
-      this.state = {term: '', results: []}
-  
-      this.handleSubmit = this.handleSubmit.bind(this)
+        super(props)
```

Comment:
If your component doesn't make use of any component-local state, you can use Functional Components. Look them up, but they are basically this:
```
function App(props) {
  return (
          <div key={i}>{r.username}: {r.content}</div>	            <div>
                <Header />
                <Navigation />
                <Section />
                <Footer />
            </div>
        )	        )
}
```